### ISSUE TITLE
Method `getEpicIssues` of `EpicsApi` should return `List<Issue>` not `List<Epic>`

### ISSUE THREAD

**Issue by hsebouai** [gitlab4j/gitlab4j-api#989]
Right now the ability to retrieve a given epic issues have the following signature:

```
public List<Epic> getEpicIssues(Object groupIdOrPath, Long epicIid) throws GitLabApiException 
```

Even if right now the serialization is working because the `Epic` model is composed of all the properties that we receive when de-serializing, It does not make sense that we return a list of epics rather that list of issues.

Furthermore, we loose all the pertinent information that an issue exposes like the `project_id, assignees, type, ...` and so on.

**Comment by jmini** [comment]
I think this is a duplicate of https://github.com/gitlab4j/gitlab4j-api/issues/729

And good news this should be fixed in `6.0.0-rc.2` and `5.3.0` that were released today.

* `5.3.0` requires Java 8 as minimal version and using a Jersey version using `javax.*` packages (for servlet, jaxrs and similar).
* `6.0.0-rc.2` requires Java 11 as minimal version and using a Jersey version using `jakarta.*` packages (for servlet, jaxrs and similar).

Please check again and re-open if you need something additional.

**Comment by hsebouai** [comment]
Thank you for the fast reply and resolution.
